catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/IngestCommand.java

```diff
@@ -589,7 +617,20 @@ private void moveToFailedIngestDirectory(File source) {
   private void buildQueue(File inputFile, ArrayBlockingQueue<Metacard> metacardQueue, long start) {
     try {
       if (includeContent) {
-        processIncludeContent(metacardQueue);
+        try (InputStream data = new FileInputStream(inputFile);
+            InputStream signature = new FileInputStream(signatureFile)) {
+          String alias = System.getProperty("org.codice.ddf.system.hostname");
+
+          if (verifier.verifyDigitalSignature(data, signature, alias)) {
+            processIncludeContent(metacardQueue);
+          } else {
+            throw new CatalogCommandRuntimeException(
+                "The provided signature was invalid for the provided data");
+          }
+        } catch (IOException | CatalogCommandRuntimeException e) {
```

Comment:
❓ Should we just be throwing this `CatalogCommandRuntimeException ` instead of throwing it, catching it, and wrapping it and rethrowing in the same exception?